README.md

```diff
@@ -1,2 +1,112 @@
-# aue
-Calculations for best- and worst-case scenario adult-use establishment distributions
+# Adult Use Establishments Proximity Analysis
+
+This repo contains code for identifying the best- and worst-case scenarios for the spatial distribution of Adult Use Establishments in NYC, given the restriction that no establishment be within 500ft of another establishment.
+
+## Background
+
+## Usage
```

Comment:
Should this and "Over view of results be a level 3 header" or should there be content under "Background"